app/views/expenses/_form.html.erb

```diff
@@ -0,0 +1,53 @@
+<div class = "bg-light p-2">
+<%= form_with(model: expense, url: user_group_expenses_path) do |form| %>
+  <% if expense.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(expense.errors.count, "error") %> prohibited this expense from being saved:</h2>
+
+      <ul>
+        <% expense.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
```

Comment:
- [ ] [OPTIONAL] it will be nice to make the error message more user-friendly
![Screenshot from 2023-09-23 20-49-41](https://github.com/Enning94/RoR-Budget-App/assets/66097371/3462917f-ff5b-42e1-bf84-21d85a64a62c)